index.html

```diff
@@ -4,170 +4,208 @@
         <button class="hero-btn" type="button">let's connect</button>
-      </div>
-      <div class="hero-social-icons">
-        <ul>
-          <li>
-            <a href="#"><i class="fa-brands fa-square-twitter"></i></a>
-          </li>
-          <li>
-            <a href="#"><i class="fa-brands fa-linkedin"></i></a>
-          </li>
-          <li>
-            <a href="#"><i class="fa-brands fa-medium"></i></a>
-          </li>
-        </ul>
-      </div>
-    </section>
+        <div class="hero-social">
+          <ul>
+            <li>
+              <a href="#"><i class="fa-brands fa-square-twitter"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa-brands fa-medium"></i></a>
+            </li>
+          </ul>
+        </div>
+      </section>
 
-    <!-- works showcase -->
-    <section class="works">
-      <div class="container grid">
-        <!-- single card -->
-        <div class="card flex">
-          <img
-            class="snapshoot"
-            src="./images/Snapshoot Portfolio.png"
-            alt=""
-          />
-          <h1 class="project-title">Tonic</h1>
-          <div class="frame">
-            <h4 class="client">CANOPY</h4>
-            <img src="./images/Counter.png" alt="" class="counter" />
-            <h4 class="role">Back End Dev</h4>
-            <img src="./images/Counter.png" alt="" class="counter" />
+      <!-- ------------works section------- -->
 
-            <p class="year">2015</p>
-          </div>
-          <div class="primary-text">
-            <p>
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required.
-            </p>
+      <section class="works">
+        <!-- ----------single card-------- -->
+        <div class="card">
+          <div class="work-col-1">
+            <img class="snapshoot" src="./images/nature.png" alt="" />
           </div>
-          <div class="tags">
-            <h3>HTML</h3>
-            <h3>CSS</h3>
-            <h3>JavaScript</h3>
+          <div class="work-col-2">
+            <h3 class="project-title">Tonic</h3>
+            <div class="frame">
+              <h4 class="client">CANOPY</h4>
+              <img src="./images/Counter.png" alt="" class="counter" />
+              <h4 class="role">Back End Dev</h4>
+              <img src="./images/Counter.png" alt="" class="counter" />
+
+              <p class="year">2015</p>
+            </div>
+            <div class="primary-text">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+            <div class="tags">
+              <h3>HTML</h3>
+              <h3>CSS</h3>
+              <h3 class="ruby">Ruby on Rails</h3>
+              <h3>JavaScript</h3>
+            </div>
```

Comment:
- **[OPTIONAL]** Its a semantic recommendation since those skills are kinda list items it's better if you use list tags here instead of headers 👍🏼